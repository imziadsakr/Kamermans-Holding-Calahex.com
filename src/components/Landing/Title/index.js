import React from 'react' ;

import {
    Box, Typography
} from "@mui/material" ;

import { makeStyles } from '@mui/styles';

const useStyles = makeStyles((theme) => ({
    root : {
        marginTop : "40px" ,
        marginBottom : "40px" ,
    } ,
    title : {
        color: theme.palette.primary.main ,
        fontSize: "60px" ,
        textAlign: "center" ,
        fontFamily: "Segoe UI, Arial, sans-serif" ,
        fontWeight : 600,
        ['@media (max-width:830px)'] : { 
           fontSize : "50px"
        },
        ['@media (max-width:690px)'] : { 
            fontSize : "40px"
        },
        ['@media (max-width:560px)'] : { 
            fontSize : "30px"
        }
    }
}));
const Title = () => {

    const classes = useStyles() ;

    return (
        <Box  className={classes.root}>
            <Box  className={classes.title}>
                Caribbean and Latin America
            </Box>
            <Box  className={classes.title}>
                Hybrid Exchange
            </Box>
        </Box>
    )
}

export default Title ;