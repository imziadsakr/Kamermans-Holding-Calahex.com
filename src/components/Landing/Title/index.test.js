import React from 'react';
import renderer from 'react-test-renderer';

import theme from '../../../utils/theme';
import { ThemeProvider } from '@mui/material';

import Title from './index';

describe("Title" , () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<ThemeProvider theme={theme}>
                        <Title />
                    </ThemeProvider>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
})
