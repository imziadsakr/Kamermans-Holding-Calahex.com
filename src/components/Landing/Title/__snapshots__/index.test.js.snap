// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<div
  className="makeStyles-root-1 MuiBox-root css-0"
>
  <div
    className="makeStyles-title-2 MuiBox-root css-0"
  >
    Caribbean and Latin America
  </div>
  <div
    className="makeStyles-title-2 MuiBox-root css-0"
  >
    Hybrid Exchange
  </div>
</div>
`;
